cclib/method/bader.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Calculation of Bader's QTAIM charges based on data parsed by cclib."""
+import random
+import numpy
+import logging
+import math
+
+from cclib.method.calculationmethod import Method
+from cclib.method.volume import electrondensity_spin
+from cclib.parser.utils import convertor
+
+# Distance between two adjacent grids (sqrt[2] or sqrt[3] for uniform Cartesian grid)
+griddist = [
```

Comment:
Can you make this private?

Reply:
Yes -- I've just updated this variable to be private. Thanks.